react/food-app/src/components/foodCard.jsx

```diff
@@ -1,60 +1,19 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState } from "react";
 
-const Foodcard = () => {
-  const [food, setFood] = useState([])
-  const [searchTerm, setSearchTerm] = useState('')
-    useEffect(() => {
-      fetch('https://pair-programming-test.s3.eu-west-2.amazonaws.com/cuisine.json').then((res) => res.json()).then((data) => setFood(data))
-    }, [])
-  
-    const  onChange = (e) => {
-      setSearchTerm(e.target.value)
-  }
-    const  onSubmit = (e) => {
-     e.preventDefault()
-  
-  }
-  
-    return (
-      <div>
-  
-        <form onSubmit={onSubmit} className='form'>
-          <label >Search For Your Favourite Food</label>
-          <input className='input-search' onChange={onChange} type='text' placeholder='UK, Fish, Pizza ...'/>
-        </form>
+const Foodcard = ({ food }) => {
+  return (
       <div className="App">
-        <ul className='food-ul' data-testid='food-card-item'>
-          {food.map((item) => {
-            if(searchTerm){
-             if(item.name.includes(searchTerm) || item.origin.includes(searchTerm)) {
-              return (
-                <li  key={item.id} className='box'>
-                <p>Food Name: {item.name}</p>
-               <p>Place Of Origin: {item.origin}</p>
-               <p>Star Rating: {item.starRating} ⭐️</p>
-               <img data-testid='food-card-item-image' className='image' src={item.image}/>
-            </li> )
-             }  
-               
-            }  else if(!searchTerm) {
-            return <li  key={item.id} className='box'>
-                <p>Food Name: {item.name}</p>
-               <p>Place Of Origin: {item.origin}</p>
-               <p>Star Rating: {item.starRating} ⭐️</p>
-               <img data-testid='food-card-item-image' className='image' src={item.image}/>
-            </li>
-  
-            }
-              
-          })}
-          </ul>
+        <li data-testid="food-card-item" key={food.id} className="box">
+          <p>Food Name: {food.name}</p>
+          <p>Place Of Origin: {food.origin}</p>
+          <p>Star Rating: {food.starRating} ⭐️</p>
+          <img
+            className="image"
+            src={food.image}
+          />
+        </li>
```

Comment:
Nice! Much neater! This is now what we call a presentational (or dumb) component, it renders according to the props it's provided. Remember this concept cus we're gunna use it a lot

Reply:
Will make sure i remember it! Thank you